pkg/security/apparmor/validate.go

```diff
@@ -46,22 +42,13 @@ type Validator interface {
 // NewValidator is in order to find AppArmor FS
 func NewValidator() Validator {
 	if err := validateHost(); err != nil {
-		return &validator{validateHostErr: err}
-	}
-	appArmorFS, err := getAppArmorFS()
-	if err != nil {
-		return &validator{
-			validateHostErr: fmt.Errorf("error finding AppArmor FS: %v", err),
-		}
-	}
-	return &validator{
-		appArmorFS: appArmorFS,
+		return nil
```

Comment:
you have totally misread my comment.

Reply:
I suggest to keep what used to be here and two lines down return and empty validator instead of `nil`

Reply:
Hi @SergeyKanzhelev, I've made changes as suggested and now all the tests are cleared.